### PR TITLE
tcc: add cross option

### PR DESCRIPTION
**Pull request by joelgibson**
Adds a --with-cross option to enable the cross compilers.

### CODE REVIEW

File: Library/Formula/tcc.rb [hunk 1]
```diff
@@ -5,11 +5,17 @@ class Tcc < Formula
   url 'http://download.savannah.gnu.org/releases/tinycc/tcc-0.9.26.tar.bz2'
   sha1 '7110354d3637d0e05f43a006364c897248aed5d0'
 
+  option "with-cross", "Build cross compilers"
+
   def install
+    opts = []
```
Comment (mistydemeo) [inline Library/Formula/tcc.rb]: How about moving the entirety of the arguments from below into a `%W[]` array, and append this onto that?

Comment (mikemcquaid) [inline Library/Formula/tcc.rb]: :+1: Also generally we use `args` for options arrays, thanks!

File: Library/Formula/tcc.rb [hunk 2]
```diff
@@ -5,11 +5,17 @@ class Tcc < Formula
   url 'http://download.savannah.gnu.org/releases/tinycc/tcc-0.9.26.tar.bz2'
   sha1 '7110354d3637d0e05f43a006364c897248aed5d0'
 
+  option "with-cross", "Build cross compilers"
```
Comment (mikemcquaid) [inline Library/Formula/tcc.rb]: For which platforms?

Comment (joelgibson) [inline Library/Formula/tcc.rb]: For anything it targets, as listed in the docs. Listing them all would be odd, as some of them are experimental. LLVM just has an option "Build all target backends"

Comment (mikemcquaid) [inline Library/Formula/tcc.rb]: Maybe `build all cross compilers`?

**Comment by joelgibson** [comment]
Thanks, added those suggestions in.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**